[zh-cn]: update the translation of 'RangeError: radix must be an integer' (#25380)
mdn/translated-content: files/zh-cn/web/javascript/reference/errors/bad_radix/index.md

---
title: "RangeError: radix must be an integer"
slug: Web/JavaScript/Reference/Errors/Bad_radix
l10n:
  sourceCommit: 6d606174faaedaa5dee7b7ebd87602cd51e5dd7e
---

{{jsSidebar("Errors")}}

> {{jsxref("Number.prototype.toString()")}} 或 {{jsxref("BigInt.prototype.toString()")}} 方法的可选 `radix` 参数被指定，但其值不在 2 到 36 之间时，会触发“radix must be an integer at least 2 and no greater than 36”JavaScript 异常。

Reviewer: ```suggestion
JavaScript 异常“radix must be an integer at least 2 and no greater than 36”会在 {{jsxref("Number.prototype.toString()")}} 或 {{jsxref("BigInt.prototype.toString()")}} 方法的可选 `radix` 参数被指定，但其值不在 2 到 36 之间时触发。
```

> ## 错误信息

Reviewer: 就翻译成“消息”吧，英文标题并没有携程“Error Message”

> ## 错误信息

Author: ```suggestion
## 消息
```

```plain

> RangeError: toString() radix argument must be between 2 and 36 (V8-based & Safari)

Reviewer: ```suggestion
RangeError: toString() radix argument must be between 2 and 36（基于 V8 & Safari）
```

> RangeError: radix must be an integer at least 2 and no greater than 36 (Firefox)

Reviewer: 看这边能不能改成这样

```suggestion
RangeError: radix must be an integer at least 2 and no greater than 36（Firefox）
```
